### PR TITLE
feat(content): Update Deep hails and ship names

### PR DESCRIPTION
**Pull request by bene-dictator**
## Summary
Adds new Deep Security hails, such as:

> We work in conjunction with the Navy to ferret out agents of base values.
> Our ships shall safeguard the Deep's hard-gained peace, prosperity, and progress.
> We advise you to liase with the Deep's autonomous security forces if you witness illegal attacks by anarchists.
> The Deep bids your enterprise prosperity throughout the commonwealth.
> The ships of the Deep work to give your ship success in these times.
> Peace, prosperity, progress.
> Please contact Deep Security if you witness the unlawful possession of sanctioned ships.

> Your hostile actions are disrupting the common good of our civilization.
> You are infracting upon the laws of the Deep.

Also adds more names to Deep Security ships, including the names of their planets and star systems and more names of intellectuals, such as:

> D.S.S. Conway
> D.S.S. Copernicus
> D.S.S. Gauss
> D.S.S. Hypatia
> D.S.S. Mendeleev
> D.S.S. Volta

> D.S.S. Valhalla
> D.S.S. Alexandria
> D.S.S. Fenrir

> D.S.S. Wezen
> D.S.S. Naos
> D.S.S. Epsilon Leonis

### CODE REVIEW

File: data/human/names.txt [hunk 1]
```diff
@@ -3420,7 +3491,7 @@ phrase "trees"
 		"Elder"
 		"Elm"
 		"Eucalyptus"
-		"Ginko"
+		"Ginkgo"
```
Comment (lumbar527) [inline data/human/names.txt:3494]: This seems to be an unrelated change.

**Review by lumbar527 — APPROVED** [review]

File: data/human/names.txt (hunk 1 above)
Comment (bene-dictator) [reply data/human/names.txt:3494]: Yeah, Ginko is a typo. Might as well change it here instead of another PR.

**Review by tibetiroka — APPROVED** [review]

File: data/human/names.txt [hunk 2]
```diff
@@ -3329,52 +3348,104 @@ phrase "intellectuals"
...
+		"Midgard"
+		"Windblain"
+		"Helheim"
+		"Alfheim"
+		"Huginn"
+		"Muninn"
+		"Fenrir"
+		"Alexandria"
+		"Leviathan"
+		"Pheonix"
```
Comment (bene-dictator) [inline data/human/names.txt:3433]:
```suggestion
		"Phoenix"
```

File: data/human/hails.txt [hunk 3]
```diff
@@ -5824,6 +5827,282 @@ phrase "friendly deep"
...
+	word
+		"Please"
+		"We advise you to"
+		"Do not hesitate to"
+	word
+		" "
+	word
+		"contact"
+		"give evidence to"
+		"liase with"
```
Comment (ziproot) [inline data/human/hails.txt:5908]:
```suggestion
		"liaise with"
```

**Review by ziproot — CHANGES_REQUESTED** [review]
GitHub actions also found a bunch of non-typos, but those are all unrelated.
(aesthetic is also an actual word, I don't know why that's being flagged)

**Comment by EjoThims** [comment]
> EDIT: aesthetic is british spelling

No, aesthetic is proper in English the world over, but US solely does also recognize the variant esthetic (and mostly even then because aestheticians and estheticians are  classified under different regulations).

Why we very suddenly have to include aesthetic in the code spell exclusions still baffles me, but just throw the relevant lines in there.

**Comment by ziproot** [comment]
Sorry about that. Merriam-Webster worded "aesthetic" as "British usually" and I found the correction in the British English to US English dictionary at the codespell repo, so I thought that was the case. When I researched further, I realized that aesthetic is more commonly used, so I've now filed a bug report to the codespell repo.

**Comment by EjoThims** [comment]
Thank you for looking into it further from the end of codespell.

:)

File: data/human/names.txt [hunk 4]
```diff
@@ -3318,8 +3334,11 @@ phrase "intellectuals"
 		"Fourier"
 		"Franklin"
 		"Fresnel"
+		"Galilei"
```
Comment (warp-core) [inline data/human/names.txt:3337]: Isn't it:
```suggestion
		"Galileo"
```

Comment (bene-dictator) [reply data/human/names.txt:3337]: Galileo's last name is Galilei, hmm... but still could get changed.

Comment (warp-core) [reply data/human/names.txt:3337]: I don't think he's really known or thought of as Gelilei.

**Comment by ziproot** [comment]
https://github.com/codespell-project/codespell/commit/7d6f3520fafa56e6b1df1e2bb533bd033218ee75

File: data/human/hails.txt [hunk 5]
```diff
@@ -5824,6 +5827,281 @@ phrase "friendly deep"
...
+		"the ones who serve "
+		"agents of "
+		"our shared "
+		"our values of "
+	word
+		""
+		"valued "
+		"invaluable "
+		"hard-fought "
+		"hard-gained "
```
Comment (ziproot) [inline data/human/hails.txt:5879]: This can lead to phrases like "our values of valued democracy" or "agents of invaluable democracy." I personally think these adjectives can be removed, but I don't know of a good solution.

File: data/human/hails.txt [hunk 6]
```diff
@@ -5824,6 +5827,281 @@ phrase "friendly deep"
...
+		"You have found yourself"
+		"You are"
+		"Your actions are"
+		"The actions of your ship are"
+	word
+		" "
+	word
+		"in breach of"
+		"in violation"
+		"in transgression"
```
Comment (ziproot) [inline data/human/hails.txt:6089]:
```suggestion
		"in violation of"
		"in transgression of"
```

File: data/human/names.txt [hunk 7]
```diff
@@ -3318,8 +3334,11 @@ phrase "intellectuals"
 		"Fourier"
 		"Franklin"
 		"Fresnel"
+		"Galileo"
```
Comment (ziproot) [inline data/human/names.txt:3337]: As this was already a point of contention, I'm fine with this not being changed. However, I personally think this should also be a last name, for consistency.

**Review by ziproot — CHANGES_REQUESTED** [review]
There were a few phrases that I found that didn't make sense. I also highlighted the name "Galileo" as I think that it should be a last name for consistency, but I understand that this was already a point of contention and am fine if this is kept the way it currently is.

File: data/human/hails.txt (hunk 5 above)
Comment (bene-dictator) [reply data/human/hails.txt:5879]: I'll remove 'agents of' instead, it should be better.

File: data/human/hails.txt [hunk 8]
```diff
@@ -5824,6 +5827,281 @@ phrase "friendly deep"
...
+		"advance"
+		"work in advancing"
+	word
+		" "
+	word
+		"" 3
+		"the Deep's "
+		"those who bring "
+		"the ones who serve "
+		"agents of "
```
Comment (bene-dictator) [inline data/human/hails.txt:5871]:
```suggestion
```

File: data/human/hails.txt (hunk 5 above)
Comment (ziproot) [reply data/human/hails.txt:5879]: "our values of" should also be removed if this is the solution
```suggestion
	word
		"" 3
		"the Deep's "
		"those who bring "
		"the ones who serve "
		"our shared "
	word
		""
		"valued "
		"invaluable "
		"hard-fought "
		"hard-gained "
```

File: data/human/hails.txt [hunk 9]
```diff
@@ -5824,6 +5827,279 @@ phrase "friendly deep"
...
+		"peace"
+		"economic prosperity"
+		"economic progress"
+		"progress"
+		"scientific progress"
+		"social cohesion"
+		"peace, prosperity, and progress"
+		"peace among the stars"
+	word
+		"."
```
Comment (ziproot) [inline data/human/hails.txt:5893]:
```suggestion
		"" 3
		"${deep adjectives}" 12
		"the Deep's "
		"the Deep's ${deep adjectives}" 4
		"those who bring "
		"those who bring ${deep adjectives}" 4
		"the ones who serve "
		"the ones who serve ${deep adjectives}" 4
		"our shared "
		"our shared, ${deep adjectives}" 4
	word
		"democracy"
		"commonwealth"
		"sovereignty"
		"autonomy"
		"prosperity"
		"peace"
		"economic prosperity"
		"economic progress"
		"progress"
		"scientific progress"
		"social cohesion"
		"peace, prosperity, and progress"
		"peace among the stars"
	word
		"."

phrase "deep adjectives"
	word
		"valued "
		"invaluable "
		"hard-fought "
		"hard-gained "
```
When you have something like "our shared" followed by an adjective like "invaluable" it becomes "our shared, invaluable" whereas when it is just followed by a word like "progress" it becomes "our shared progress." This allows the combinations you want while being, as far as I can tell, grammatically correct. I looked at the other combinations and after this there shouldn't be any problems as far as I can see.

Comment (ziproot) [inline data/human/hails.txt:5893]: Apologies for the double comment. I found a cleaner solution after posting that solves the same issue: the phrase "our shared valued democracy" requires a comma after "shared" while the phrase "our shared democracy" doesn't.
```suggestion
		"our shared% "
	word
		""
		"%valued "
		"%invaluable "
		"%hard-fought "
		"%hard-gained "
	word
		"democracy"
		"commonwealth"
		"sovereignty"
		"autonomy"
		"prosperity"
		"peace"
		"economic prosperity"
		"economic progress"
		"progress"
		"scientific progress"
		"social cohesion"
		"peace, prosperity, and progress"
		"peace among the stars"
	word
		"."
	replace
		"% %" ", "
		"% " ""
```

File: data/human/hails.txt [hunk 10]
```diff
@@ -5824,6 +5827,282 @@ phrase "friendly deep"
...
+		"the Deep's "
+		"those who bring "
+		"the ones who serve "
+		"our shared% "
+	word
+		""
+		"%valued "
+		"%invaluable "
+		"%hard-fought "
+		"%hard-gained "
```
Comment (bene-dictator) [inline data/human/hails.txt:5877]:
```suggestion
		"%cherished "
		"%precious"
```

File: data/human/hails.txt [hunk 11]
```diff
@@ -5824,6 +5827,282 @@ phrase "friendly deep"
...
+		"advance"
+		"work in advancing"
+	word
+		" "
+	word
+		"" 3
+		"the Deep's "
+		"those who bring "
+		"the ones who serve "
+		"our shared% "
```
Comment (bene-dictator) [inline data/human/hails.txt:5871]:
```suggestion
		"our shared% "
		"our"
```

File: data/human/hails.txt [hunk 12]
```diff
@@ -5824,6 +5827,282 @@ phrase "friendly deep"
...
+		"autonomy"
+		"prosperity"
+		"peace"
+		"economic prosperity"
+		"economic progress"
+		"progress"
+		"scientific progress"
+		"social cohesion"
+		"peace, prosperity, and progress"
+		"peace among the stars"
```
Comment (bene-dictator) [inline data/human/hails.txt:5891]:
```suggestion
		"peace among the stars"
		"freedoms"
```

File: data/human/hails.txt [hunk 13]
```diff
@@ -5824,6 +5827,285 @@ phrase "friendly deep"
...
+		"freedoms"
+	word
+		"."
+	replace
+		"% %" ", "
+		"% " ""
```
[the remark this answers is not in the document]
Comment (bene-dictator) [reply data/human/hails.txt:5899]: No worries. I wish that hail testing PR was merged, it'd make life so easy for us.

**Comment by bene-dictator** [comment]
We have no problem with using the planet and system names for Deep ships, right? It just occurs to me that it's not something you see on any NPC ship apart from mission ones, but I figure the Deep is more focused on their planets and would honour them by naming their ships after them.

Similarly, why doesn't the Republic have ships named for planets or systems? Too complicated?

File: data/human/hails.txt [hunk 14]
```diff
@@ -5824,6 +5827,285 @@ phrase "friendly deep"
...
+		"wealth"
+	word
+		" "
+	word
+		"in these times"
+		"in our space"
+		"throughout our space"
+		"throughout the commonwealth"
+	word
+		"."
```
Comment (ziproot) [inline data/human/hails.txt:6028]:
```suggestion
	word
		"."
	replace
		"% bid" " bids"
		"% wish" " wishes"
		"% offer" " offers"
		"% work to give" " works to give"
		"% ensure" " ensures"
```

File: data/human/hails.txt [hunk 15]
```diff
@@ -5824,6 +5827,285 @@ phrase "friendly deep"
...
+		"We"
+		"The citizens of the Deep"
+		"The ships of the Deep"
+	word
+		" "
+	word
+		"bid"
+		"wish"
+		"offer"
+		"work to give"
```
Comment (ziproot) [inline data/human/hails.txt:6003]:
```suggestion
		"work to give"
		"ensure"
```

File: data/human/hails.txt [hunk 16]
```diff
@@ -5824,6 +5827,285 @@ phrase "friendly deep"
...
+		"wealth"
+	word
+		" "
+	word
+		"in these times"
+		"in our space"
+		"throughout our space"
+		"throughout the commonwealth"
+	word
+		"."
```
Comment (ziproot) [inline data/human/hails.txt:5989]:
```suggestion
```

**Review by ziproot — APPROVED** [review]
At this point, I don't see any grammatical or spelling errors that would appear in game, so I am approving this PR. There is still my outstanding comment from earlier about a ship being named "Galileo" when I think it should be named "Galilei" but as I said in that comment, that was a point of contention and remains optional. I do have some suggestions on refactoring two phrases that I think are mostly the same so that they can be merged into one, using `replace` syntax, but as this is a refactor, I am offering my approval now, as I am fine with this PR whether or not my changes are merged.

**Comment by bene-dictator** [comment]
Better to merge it into one hail to keep it from being more common than other Deep Sec hails as well as shorten the file a little bit. Thank you.

**Review by ziproot — APPROVED** [review]
Looks good now. Thank you for your patience.

File: data/human/hails.txt [hunk 17]
```diff
@@ -5824,6 +5827,257 @@ phrase "friendly deep"
...
+		"the Deep's autonomous security forces" 6
+		"our security forces" 6
+		"our officials" 6
+		"our agents" 6
+		"our officers" 6
+		"Deep authorities" 6
+	word
+		" "
+	word
+		"in the event of"
```
[the remark this answers is not in the document]
Comment (bene-dictator) [reply data/human/hails.txt:5937]: I'll remove "anti-Deep forces" and "anti-Republic forces".

File: data/human/hails.txt [hunk 18]
```diff
@@ -5824,6 +5827,257 @@ phrase "friendly deep"
...
+		"your enterprise"
+	word
+		" "
+	word
+		"peace, prosperity, and progress"
+		"peace"
+		"prosperity"
+		"progress"
+		"success"
+		"wealth"
```
[the remark this answers is not in the document]
Comment (bene-dictator) [reply data/human/hails.txt:5985]: Perhaps replace it with "good fortune"?

File: data/human/hails.txt [hunk 19]
```diff
@@ -5824,6 +5827,257 @@ phrase "friendly deep"
...
+		"in the event of"
+		"if you witness"
+		"if you have been the target of"
+		"if you have been affected by"
+	word
+		" "
+	word
+		"piracy"
+		"anti-Deep forces"
+		"anti-Republic forces"
```
Comment (bene-dictator) [inline data/human/hails.txt:5946]:
```suggestion
```

File: data/human/hails.txt (hunk 18 above)
Comment (bene-dictator) [inline data/human/hails.txt:5985]:
```suggestion
		"good fortune"
```

File: data/human/hails.txt [hunk 20]
```diff
@@ -5824,6 +5827,257 @@ phrase "friendly deep"
...
+		"your ship"
+		"your vessel"
+		"your enterprise"
+	word
+		" "
+	word
+		"peace, prosperity, and progress"
+		"peace"
+		"prosperity"
+		"progress"
```
Comment (bene-dictator) [inline data/human/hails.txt:5983]:
```suggestion
		"security"
```
As 'progress' does not really stand by itself in this context.

File: data/human/hails.txt [hunk 21]
```diff
@@ -5824,6 +5827,257 @@ phrase "friendly deep"
...
+		"success"
+		"wealth"
+	word
+		" "
+	word
+		"in these times"
```
Comment (bene-dictator) [inline data/human/hails.txt:5989]:
```suggestion
```
Alternatively, remove "in our space".

File: data/human/hails.txt (hunk 18 above)
[the remark this answers is not in the document]
Comment (Anarchist2) [reply data/human/hails.txt:5985]: "prosperity" covers the idea fine.
